Makefile.toml

```diff
@@ -0,0 +1,9 @@
+[tasks.build]
+command = "cargo"
+args = ["build"]
+dependencies = ["format", "clippy"]
+
+[tasks.test]
```

Comment:
I though about it a bit but I didn't find the way to do it fast. I think we would need to find an image in CircleCI that already has cargo-make, clippy and rustfmt because if we need to install it in each run that will end being very slow. Do you know a solution?

Reply:
we can cache the globally installed dependencies as a separate cache key. I need to check CircleCi more thoroughly though.
Our project is still small so on Travis we always install `cargo-make` initially and then it installs clippy & rustfmt.

This one has rustfmt https://hub.docker.com/r/circleci/rust/dockerfile
and I think we can add `clippy` with `add component`, but I can try it and tell you.